tests/unit/crc/main.c

```diff
@@ -9,8 +9,19 @@
 #include "../../../lib/os/crc8_sw.c"
 #include "../../../lib/os/crc16_sw.c"
 #include "../../../lib/os/crc32_sw.c"
+#include "../../../lib/os/crc32c_sw.c"
 #include "../../../lib/os/crc7_sw.c"
 
+void test_crc32c(void)
+{
+	uint8_t test1[] = { 'A' };
+	uint8_t test2[] = { '1', '2', '3', '4', '5', '6', '7', '8', '9' };
+	uint8_t test3[] = { 'Z', 'e', 'p', 'h', 'y', 'r' };
+
+	zassert_equal(crc32_c(0, test1, sizeof(test1), 1, 1), 0xE16DCDEE, NULL);
+	zassert_equal(crc32_c(0, test2, sizeof(test2), 1, 1), 0xE3069283, NULL);
+	zassert_equal(crc32_c(0, test3, sizeof(test3), 1, 1), 0xFCDEB58D, NULL);
```

Comment:
Not sure if part of regular test cases, but would be great if there is a test case where first_pkt = 0 to increase coverage.

Reply:
Thanks for pointing it out. It's a shame i missed it. Done now.